clash-ghc/src-ghc/Clash/GHC/LoadModules.hs

```diff
@@ -501,7 +501,7 @@ loadModules startAction useColor hdl modName dflagsM idirs = do
                   No top-level function called 'topEntity' or 'testBench' found,
                   nor any function annotated with a 'Synthesize' or 'TestBench'
                   annotation. If you want to synthesize a specific binder in
-                  #{show modName}, use '-main-is=myTopEntity'.
+                  #{show modName}, use '-main-is myTopEntity'.
```

Comment:
Oof, telling users to do a thing that doesn't work is the worst kind of helpful message :see_no_evil:

Reply:
Or is it the best kind of unhelpful message? :wink:

Reply:
Hah